goodbye-graffiti-agent/README.md

```diff
@@ -0,0 +1,56 @@
+1. You must have at least one API key associated with your project. Go to the Google Maps Platform > Credentials [page](https://console.cloud.google.com/project/_/google/maps-apis/credentials?utm_source=Docs_CreateAPIKey&utm_content=Docs_static-maps-backend&_gl=1*8gesr0*_ga*MTMxNzQwMTEyNS4xNjkyMDE1OTQ1*_ga_NRWSTWS78N*MTY5MjAxNTk0Ni4xLjEuMTY5MjAxNzI2MS4wLjAuMA..) and create an [API key](https://developers.google.com/maps/documentation/maps-static/get-api-key#creating-api-keys).  
+1. Open file `index.js` located under `maps-function` folder. You must include your API key with both the Geocoding API and Maps Static API requests. Replace `YOUR_API_KEY` with your API key. 
+1. Enable the Cloud Functions, Cloud Build, Artifact Registry, Cloud Run, Dialogflow API and Cloud Logging APIs
+1. Install and initialize the [gcloud CLI](https://cloud.google.com/sdk/docs/install).
+
+## Deploy the Cloud Function
+To deploy the function, run the `gcloud functions deploy` command in the `maps-function` directory:
+`gcloud functions deploy lookupPlace --runtime=nodejs20 --region=REGION --source=. --entry-point=lookupPlace --trigger-http --allow-unauthenticated`
+
+Replace REGION with the name of the Google Cloud region where you want to deploy your function (for example, us-west1).
+The optional `--allow-unauthenticated` flag lets you reach your function without authentication.
+
+After the function deploys, note the url property from the output of the gcloud functions deploy command, or retrieve it with the command `gcloud functions describe lookupPlace --region REGION`. Replace REGION with the name of the Google Cloud region where you deployed your function. You will need the url when configuring the webhook in Dialogflow CX. 
+
+## Create the agent and configure the webhook
+1. In your browser, navigate to the [Dialogflow CX console](https://dialogflow.cloud.google.com/cx/projects)
+1. Create a new agent (select the option **Build your own agent**). You must create the agent in the **global** region as this is the only region where the Phone Gateway integration is currently supported. You cannot change region once the agent is deployed. 
+1. [Restore](https://cloud.google.com/dialogflow/cx/docs/concept/agent#export) the Goodbye Graffiti agent exported in the JSON package file format (agent.zip) 
+1. Once you have successfully restored the agent, under the **Manage** tab open the definition of the **lookup-place** webhook
+1. Enter your function URL as the webhook URL and click Save.
+
+![Dialogflow CX Webhook](images/webhook.png)
```

Comment:
Should the `YOUR-DFCX-INSTANCE` placeholder text in this screenshot instead be `YOUR-CLOUD-RUN-ENDPOINT` or similar?